modules/KalturaSupport/components/cvaa/cvaa.js

```diff
@@ -228,6 +229,11 @@
                         _this.initPreviewUpdate(_this.getCurrentPreset(_this.cvaaSettings.currentPreset));
                         _this.saveCvaaSettings();
                     }
+
+                    if(event.which === 32 || event.which === 13){
```

Comment:
same logic on line 226, we could move this condition to separate method

Reply:
@semarche-kaltura 
not exactly 
line 226 covers click, enter and space, while here - I want to close the screen only if user uses the keyboard. If he used a mouse-click I do not want to close the screen.